src/@commonsku/styles/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import styled from "styled-components";
+import { ChevronIcon } from "./icons";
+import colors from "./colors";
+
+//Styles Begin
+const ScrollControl = styled.tr`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+`;
+
+const Dot = styled.button<{ $bold?: boolean; $dotColor: string }>`
+  &&& {
+    width: ${(props) => (props.$bold ? "13px" : "8px")};
+    height: ${(props) => (props.$bold ? "13px" : "8px")};
+    border-radius: 100%;
+    padding: 0;
+    border: none;
+    cursor: pointer;
+    background-color: ${(props) => props.$dotColor};
+  }
+`;
+
+const ArrowButtons = styled(ChevronIcon)<{ $off?: boolean }>`
```

Comment:
Small nitpick but since `ArrowButtons` is a single button, it should be called `ArrowButton`.